vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -1,53 +1,67 @@
 
 ## Logging into your Keycloak Instance
 
-In an Entando deployment Keycloak is protected by a Secret deployed in your Kubernetes instance. To get the default admin credentials you can query Kubernetes for the secret with this command:
+Keycloak is protected by a Secret deployed to your Entando Kubernetes instance. You can query Kubernetes for the Secret's default admin credentials, substituting your environment's namespace and Secret name:
 
 ```
-kubectl get secret quickstart-kc-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
+ent kubectl get secret quickstart-kc-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
 ```
 
-Update the namespace and secret name to match your environment. If you're not sure of the secret name you can run
+ To find the Secret name, run
 ```
 kubectl get secrets -n <namespace>
 ```
-And search for the secret that ends in `kc-admin-secret`
+and search for the Secret that ends in `kc-admin-secret`.
 
 ## Authentication
-In Entando 6 all authentication goes through Keycloak. This ensures that a micro frontend can call a microservice with a token that is available on the client.
+Beginning with Entando 6, all authentication is powered by Keycloak. This ensures that a micro frontend can call a microservice with a token available to the client.
 
 ![Init Containers Screenshot](./img/keycloak-arch-high-level.png)
 
-By using Keycloak as a central point of authentication the Entando architecture is able to provide a single unified view of identity to the entire architecture. With this architecture Entando becomes more portable and can be integrated into other IDPs without changes to the source. Keycloak acts as an  abstraction to the underlying IDP.
+The Entando architecture implements Keycloak as a central point of authentication. A single, unified view of identity increases Entando's portability. Entando can also integrate into other IDPs without modifying the source, because Keycloak acts as an abstraction to the underlying IDP.
 
 ## Authorization
 
 ### Plugins/Microservices
-Authorization for the microservices comes from clients and roles in keycloak. The authorizations are stored in the JWT token and are available to the services when invoked.
+Keycloak authorizes microservices using clients and roles. Authorizations are stored in a JWT token and available to services when invoked.
 
-### Core
-Authorization for the entando-core and WCMS is provided by the Entando user management database. When a user is authenticated to the entando-core  a copy of that user is added to the Entando user management database.  That copy is made in support of the authorization flow noted below.
+To setup permissions to configure the service:
```

Comment:
"These are the steps to grant a user one or more roles for a specific client. Note: when a microservice is installed in Entando, a corresponding client (and set of roles) is created per its [plugin definition](docs/ecr/ecr-bundle-details.html#plugin) "  

@Lyd1aCla1r3 I'm pretty sure my link is wrong but you get the idea...

Reply:
should the link go here: https://developer.entando.com/next/tutorials/backend-developers/generate-microservices-and-micro-frontends.html#overview or are we talking a repo of yaml files? i know better than to link to GitHub without making sure it's front-facing and not subject to change with revs...

Reply:
sorry, @nshaw